### PR TITLE
Remove pill when country name is a part of zone name

### PR DESCRIPTION
**Pull request by silkeholmebonnen**
## Issue

[Linear 3341](https://linear.app/electricitymaps/issue/ELE-3341/remove-the-country-pill-when-country-name-is-part-of-the-zone)

## Description

We only want to show the country name in the pill if the country name is not a part of the zone name. E.g. we don't want to have the pill **Denmark** when the zone name is _Eastern Denmark_, but we do want the pill **Italy** when the zone name is _Sicily_.

### Preview

image

image

### Double check

- [X] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: web/src/features/panels/zone/ZoneHeaderTitle.tsx [hunk 1]
```diff
@@ -22,10 +22,10 @@ export default function ZoneHeaderTitle({
 }: ZoneHeaderTitleProps) {
   const { __ } = useTranslation();
   const title = getZoneName(zoneId);
-  const isSubZone = zoneId.includes('-');
   const returnToMapLink = createToWithState('/map');
   const countryName = getCountryName(zoneId);
   const disclaimer = getDisclaimer(zoneId);
+  const showCountryPill = zoneId.includes('-') && !title.includes(countryName);
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/features/panels/zone/ZoneHeaderTitle.tsx:28]: I disagree, Mr. Robot...

[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/features/panels/zone/ZoneHeaderTitle.tsx:28]: This makes no sense, why is not suddenly trimming things? 😅

**Comment by VIKTORVAV99** [comment]
> @madsnedergaard Do you think I should make tests for this? And do you think I should implement the suggestions from the bot?

The bot is kind of crazy at times, don't trust it! 😅
On a more serious note, the some of the bots suggestions make sense (maybe not in this PR) but you are almost always better of implementing them yourself than accepting it's suggestions.

**Comment by madsnedergaard** [comment]
> > @madsnedergaard Do you think I should make tests for this? And do you think I should implement the suggestions from the bot?
> 
> The bot is kind of crazy at times, don't trust it! 😅 On a more serious note, the some of the bots suggestions make sense (maybe not in this PR) but you are almost always better of implementing them yourself than accepting it's suggestions.

Yeah, it's also weird how the "code feedback" is different from the comments. In this case, the suggestion of "US could be part of HoUSton" actually makes sense in general (I checked all zone names and it doesn't matter for us though), but the comment suggestions are rubbish :)

**Review by madsnedergaard — APPROVED** [review]
Looks good to me! :)

**Pull request merged**